tests/behat/behat-features/replication-aditions.feature

```diff
@@ -0,0 +1,227 @@
+    And I select "-Cooking" from "edit-field-channel"
+    And I select "published" from "moderation_state[0]"
+
+    # Add teaser
+    And I press "field_teaser_media_entity_browser_entity_browser"
+    Then I wait for AJAX to finish
+    Then I wait for ".views-row:nth(3)" in entity browser "image_browser"
+    And I click on unused media in "image_browser"
+    And I click on "#edit-submit" in entity browser "image_browser"
+    And I wait for entity browser "image_browser" to close
+    Then I wait for AJAX to finish
+    Then Save "[name='field_teaser_media[target_id]']" value as currently selected
+
+    # Save and publish article.
+    And I select "Published" from "moderation_state[0]"
+    And I press "Save as"
+    And I wait for the page to be loaded
+
+    # Replicate
+    Given I open the satellite
+    When I run drush cppull
+    Then drush output should contain "Content of remote /Contentpool/ has been replicated successfully."
+    And I am logged in as a user with the "administrator" role
+
+    # Check media and article are replicated
+    And I am on "/"
+    Then I should see the text "BEHAT: Media 2"
+
+    # Check media is replicated
+    And I am on "admin/content/media"
+    Then I should see title of currently selected media
+    And I should get a 200 HTTP response
+
+  @javascript @replication-aditions-media-edited
+  Scenario: Replication works for edited media
+    # Remove reference media and replicate
+    Given I am logged in to contentpool
+    And I visit path "/" on contentpool
+    And I click "BEHAT: Media 2"
+    And I wait for the page to be loaded
+    Then I should get a 200 HTTP response
+    Then I should see the text "Edit"
+    And I click "Edit" in local tasks
+    And I wait for the page to be loaded
+    Then Save "[name='field_teaser_media[target_id]']" value as currently selected
+    And I press "edit-field-teaser-media-current-items-0-remove-button"
+    And I wait for AJAX to finish
+    Then "[name='field_teaser_media[target_id]']" value is empty
+    Then I wait for AJAX to finish
+
+    # Save and publish article.
+    And I select "Published" from "moderation_state[0]"
+    And I press "Save as"
+    And I wait for the page to be loaded
+
+    # Replicate
+    Given I open the satellite
+    When I run drush cppull
+    Then drush output should contain "Content of remote /Contentpool/ has been replicated successfully."
+    And I am logged in as a user with the "administrator" role
+
+    # Check article replicated
+    And I am on "/"
+    Then I should see the text "BEHAT: Media 2"
+
+    # Edit media
+    Given I visit path "admin/content/media" on contentpool
+    And I edit last media
+    Then I wait for the page to be loaded
+    And I fill in "name[0][value]" with "BEHAT: Image media edited"
+    And I press "Save and keep published"
+
+    # Edit article
+    And I visit path "/" on contentpool
+    And I click "BEHAT: Media 2"
```

Comment:
a bit confusing that article and media item are labelled the same. Make call it "BEHAT: Media article 2".